package.json

```diff
@@ -17,6 +17,7 @@
     "lerna": "^6.6.1"
   },
   "dependencies": {
-    "dotenv": "^16.0.3"
+    "dotenv": "^16.0.3",
+    "material-icons": "^1.13.11"
```

Comment:
I think something went wrong with your install. This should be installed inside the lms project, not on the root. Please move this to the other package file.

Reply:
Moving the material icons from `package.json` to `packages/lms/package.json`
Should the dotenv also be moved?

Reply:
And should i remove this `"@mollify/lms": "^0.2.11",` from the `packages/lms/package.json` ?

Reply:
Keep dotenv, and remove @mollify/lms